### DISCUSSION TITLE
烤鱼 | /cn/2022/05/grill-fish/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1469] [Comments]
一条白斑狗鱼
一条淡水石首鱼
偏偏是不爱吃鱼的
一头黄老师
如同天选之子
以一敌二
纷纷鳞甲飞
战斗了大半夜
终于用拆炸弹的刀取了敌方的狗头鱼首
&hellip;

https://yihui.org/cn/2022/05/grill-fish/

**Comment by liechi** [comment]
你在江湖上人称“谢老师”吗？还是为了跟“黄大厨”对仗才这么写？我的朋友要是这么叫我，友谊的小船估计要翻，哈哈哈。

狗鱼味道怎么样？我最近很想吃猫鱼 (cat fish; 鲶鱼)，可惜日本海鱼好找，河鱼难。水煮鱼片真是久违了。

**Reply by yihui** [reply]
不是为了对仗，我在本地生活圈内经常被称作“谢老师”，但这对我来说也是个谜，因为我就没当过一天老师，不知道为什么我在他们眼中看起来像老师。这个称呼的来源在这里：https://yihui.org/cn/2005/05/teacher-xie/ 从那时候我的同学都这样叫我了。09 年来美国之后没人知道这个掌故，所以就没人这样叫了，但有一天有个同学又突然发明了这个称呼，我问她为什么，她说你看起来就像个老师啊。于是这个称呼在美国又死灰复燃。等我毕业搬家后，这个故事在新的城市又重演了一遍。这个称呼也在这里生根发芽了，我的好朋友们都习惯了这样叫我，在他们面前连我自己都习惯了这样自称。我们这几个朋友才是真的老师，我是混在里面的冒牌货。

狗鱼的味道很好，不过说实话那并非狗鱼本身好，而是四川火锅调味料好（好像是“桥头”吧）。黄大厨家经常买很多美味的川菜调料。我把狗鱼剁成小块后在烧烤架上烤一烤，烤的焦干一些（这是重点，不然鱼总是水唧唧的），然后下锅用火锅调料和各种配菜一炒，不输国内的烤鱼。

鲶鱼在我们这里的亚洲超市有无穷的供应。土腥味有点重，也需要重口味的调料来做。鲶鱼的肉比较紧实，我自己还比较喜欢。平时我们吃水煮鱼片都是用冷冻鱼肉，这边最常见的是虎头鲨（Swai）和罗非鱼（Tilapia）。虎头鲨的肉很嫩，更适合做水煮鱼。罗非鱼我觉得不是很好吃。前天我刚剁鱼泥做了鱼丸，味道和口感都相当不错，这是湖北的一道年菜。

**Reply by liechi** [reply]
把鱼肉烤干，再用火锅底料焖回去！我好想瞬间理解了烤鱼的要点，顺便回忆起了久违的口感和香味。
